domains/ajink.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ajinkgupta",
+        "email": "guptaajink72@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=ajink.is-a.dev,43c43224ba9d86055dcd"
```

Comment:
We do not support Vercel TXT verification. Please use a different provider if you want to use this domain.